third_party/terraform/utils/bootstrap_utils_test.go

```diff
@@ -0,0 +1,105 @@
+package google
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"testing"
+
+	"google.golang.org/api/cloudkms/v1"
+)
+
+/**
+* bootstrapKMSkey will return a KMS key that can be used in tests that are
+* testing KMS integration with other resources.
+*
+* This will either return an existing key or create one if it hasn't been created
+* in the project yet. The motivation is because keyrings don't get deleted and we
+* don't want a linear growth of disabled keyrings in a project. We also don't want
+* to incur the overhead of creating a new project for each test that needs to use
+* a KMS key.
+**/
+var SharedKeyRing = "tftest-shared-keyring-1"
+var SharedCyptoKey = "tftest-shared-key-1"
+
+type bootstrappedKMS struct {
+	*cloudkms.KeyRing
+	*cloudkms.CryptoKey
+}
+
+func bootstrapKMSKey(t *testing.T) bootstrappedKMS {
```

Comment:
nit: This feels like something we'd have as the only exported function in a separate package if we used more than 1 package, so I lean slightly towards capitalising this.